lib/index.ts

```diff
@@ -113,7 +107,7 @@ function getRawProcessList(
     requestInProgress = true;
     native.getProcessList((processList: IProcessInfo[]) => {
       queue.forEach(r => {
-        r.callback(filter(r.rootPid, processList, MAX_FILTER_DEPTH));
+        r.callback(transform(r.rootPid, processList));
```

Comment:
I think filter is more accurate here as it's filtering to only children of `rootPid`?

Reply:
In this context, the function is an arbitrary transformation from the entire raw list to the result type. Even if those functions happens to omit part of the input in addition to the re-ordering and restructuring they do, I'd argue they do not represent a filter in the normal use of that word.

Some of this feels a little moot, though: these would be better represented by binding the transform in the callback, and then there's no need for general typing at all. Was trying to keep changes limited to straightforward "make it typecheck", but maybe that was a mistake.

This would just become:
```ts
queue.forEach(r => r.callback(processList));
```

invoked as:
```ts
getRawProcessList(processTreeRequestQueue, flags,
  list => callback(buildProcessTree(rootPid, list, MAX_DEPTH)));
```